pkg/neonvm/controllers/vm_controller.go

```diff
@@ -165,6 +161,17 @@ func (r *VMReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
+	// examine cpuScalingMode and set it to the default value if it is not set
+	if vm.Spec.CpuScalingMode == nil || *vm.Spec.CpuScalingMode == "" {
```

Comment:
question: `*vm.Spec.CpuScalingMode == ""` can't happen, right?

Reply:
Yes, a leftover from the field not being valiated